policies/otc-policies.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--#include virtual="/inc/head.shtml" -->
+
+<body>
+<!--#include virtual="/inc/banner.shtml" -->
+
+<div id="main">
+  <div id="content">
+    <div class="blog-index">
+      <article>
+        <header>
+          <h2>OpenSSL Technical Policies</h2>
+          <h5>
+            First issued 30th September 2020<br/>
+            Last modified 30th September 2020
+          </h5>
+        </header>
+
+        <div class="entry-content">
+
+          <p>This document lists the technical policies and procedures established
+          by the OTC in accordance with the <a href="omc-bylaws.html">project bylaws</a>
+          and the requirements specified by the OMC.</p>
+
+          <h3><a name="voting-procedure">Voting Procedure</a></h3>
+
+          The following implementary regulations complement the
```

Comment:
implementary? No one says this word. I'd just drop it. Or (alternatively) replace with "implementation regulations"

Reply:
I got it from an online dictionary :-). Just dropped it, see d54a084af7aec9c19628f7c755abf3589a7ecb02.